submissions/kasionio/html-css-cinema-booking/index.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html lang="en">
+   <head>
+      <meta charset="UTF-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <link rel="stylesheet" type="text/css" href="index.css">
+      <title>Cinema booking</title>
+   </head>
+   <body>
+      <header class="header">
+         <h1 class="header_main">Sex education</h1>
+      </header>
+      <main class="main">
+         <div class="left-poster poster-container">
+         </div>
```

Comment:
Use multiply background images instead of empty `div` [Read more](https://developer.mozilla.org/en-US/docs/Web/CSS/background-image). Also, you can use pseudoelements for it)

Reply:
Thanks a lot for this information! I didn't know it yet.